core/src/main/scala/akka/kafka/scaladsl/Consumer.scala

```diff
@@ -171,5 +172,9 @@ object Consumer {
     }
   }
 
+  def committablePartitionedSource[K, V](settings: ConsumerSettings[K, V]):
+    Source[(TopicPartition, Source[CommittableMessage[K, V], Control]), Control] = {
```

Comment:
doesn't this open up for the same kind of problems that we originally saw with `groupBy` emitting sources. @drewhk?